### DISCUSSION TITLE
SX1262 FHSS

### DISCUSSION THREAD

**Discussion by AlbertoManske** [jgromes/RadioLib#891] [General]
Hi, I'm doing some tests with the SX1262, I saw in the datasheet that it supports FHSS, do you happen to want to add its lib?

**Comment by jgromes** [comment]
It's been on my TODO list for a while, but not a high priority sicne I never had any use case for it.

Also - this is not really an issue (and completely ignores the issue template), moving to discussion.

**Comment by AlbertoManske** [comment]
Thanks for the feedback, sorry for putting it in the wrong place, I'll be following, and congratulations on the excellent work!
